.travis.yml

```diff
@@ -1,22 +1,25 @@
 language: php
 
-dist: precise
+dist: xenial
 
 sudo: false
 
+services:
+  - postgresql
+
 matrix:
   include:
-    - php: 5.3
-      env: DB=PGSQL CORE_RELEASE=3.4
-    - php: 5.4
-      env: DB=PGSQL CORE_RELEASE=3.5
-    - php: 5.5
+    - php: 5.6
       env: DB=PGSQL CORE_RELEASE=3.6
+    - php: 5.6
+      env: DB=PGSQL CORE_RELEASE=3.7
+    - php: 7.1
+      env: DB=PGSQL CORE_RELEASE=3.7
     - php: 5.6
       env: DB=PGSQL CORE_RELEASE=3
-    - php: 7.0
+    - php: 7.2
       env: DB=PGSQL CORE_RELEASE=3
-    - php: 7.1
+    - php: 7.3
```

Comment:
@maxime-rainville I think it's less important to test 7.2 / 7.3 than it is to test 7.4, so maybe either swap one of them or add another entry?